optimum/intel/neural_compressor/quantization.py

```diff
@@ -309,10 +310,14 @@ def quantize(
             self._quantized_model = convert_to_quantized_model(
                 self._original_model, quantization_config, device=quantization_config.device
             )
-
+            quantization_config.remove_redundant_parameters()
+            self._quantized_model.config.quantization_config = quantization_config
```

Comment:
currently a `quantization_config.json` file is expected to load ITREX model https://github.com/huggingface/optimum-intel/blob/v1.17.2/optimum/intel/neural_compressor/modeling_base.py#L186 why should save it in the main config ?